dhall-toml/src/Dhall/Toml.hs

```diff
@@ -180,5 +310,28 @@ dhallToTomlMain = do
 
 
 tomlToDhallMain :: IO ()
-tomlToDhallMain = putStrLn "not implemented"
+tomlToDhallMain = do
+    text <- Text.IO.getContents
+    toml <- case Toml.Parser.parse text of
+        Left tomlErr -> throwIO (InvalidToml tomlErr)
+        Right toml -> return toml
+    args <- System.Environment.getArgs
+    schemaFile <- case args of
+        [] -> fail "schema not provided"
+        schemaFile:[] -> return schemaFile
+        _ -> fail "too many agrgs"
```

Comment:
I think it should be a priority to use a proper options parser here (`optparse-applicative`) so potential users aren't discouraged by a lack of CLI usage info.